components/Dialogs/examples/DialogsWithEmphasisButtonExampleViewController.swift

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+import MaterialComponents.MaterialButtons
+import MaterialComponents.MaterialDialogs
+import MaterialComponentsBeta.MaterialContainerScheme
+import MaterialComponentsBeta.MaterialDialogs_Theming
+
+class DialogsWithEmphasisButtonExampleViewController: UIViewController {
+
+  private let materialButton = MDCButton()
+  var containerScheme = MDCContainerScheme()
+
+  override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+    super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+    containerScheme.colorScheme = MDCSemanticColorScheme(defaults: .material201804);
```

Comment:
I think we can delete this line; did you find that it was necessary?

Reply:
Good catch. It's not necessary.